lib/destack/rollup.config.js

```diff
@@ -27,8 +25,8 @@ const browserConfig = {
       include: /\.[jt]sx?$/, // default, inferred from `loaders` option
       exclude: /node_modules/, // default
       sourceMap: false, // default
-      minify: process.env.NODE_ENV === 'production',
-      target: 'es2017', // default, or 'es20XX', 'esnext'
+      minify: true,
+      target: 'es2020', // default, or 'es20XX', 'esnext'
```

Comment:
I'm working on Github actions to automatically run tests with different node.js versions. I'll also manually test it with at least Node 10 right before merging the PR.

For now, I think the target be `node10` to make Destack available to previous versions of Node and Next.js.

Reply:
That's very strange.

Did a quick check with esbuild on Github and I can't find anywhere to explicitly check for specific node versions. I guess that means parse the version from the string eg. `nodex.y.z`

Reply:
Turns out `node10` is the default target for esbuild so it should be fine [esbuild/scripts/esbuild.js](https://github.com/evanw/esbuild/blob/3431cc835b6dcf8f6172665a68b1d9c8b80fb147/scripts/esbuild.js#L11).

Reply:
Great!